### PR TITLE
[Frontend] Feature/demo - pt1

### PR DESCRIPTION
**Pull request by ryardley**
- [x] Can demonstrate pegging ETH -> cETH
- [x] Can demonstrate adding liquidity cLINK -> ROWAN
- [x] Can demonstrate swap cUSDC -> cETH
- [x] Can demonstrate unpegging cETH -> ETH 
- [x] Temporarily set every token to 18 decimals
- [x] Added helper methods to AssetAmount
- [x] Fix images unbalanced in liquidity pool page
- [x] Return user to listing after pegging and unpegging

TODO After merge on a new branch 

- [ ] Token labels must be `cXYZ` everywhere
- [ ] Confirmation screen after add liquidity is missing token images
- [ ] Need to check values in confirmation screen that they are not hard coded
- [ ] Merge Keplr branch

### CODE REVIEW

File: ui/app/src/components/confirmationDialog/AnimatedConfirmation.vue [hunk 1]
```diff
@@ -49,6 +49,8 @@
 import { defineComponent } from "vue";
 import Loader from "@/components/shared/Loader.vue";
 import SifButton from "@/components/shared/SifButton.vue";
+import { computed } from "@vue/reactivity";
```
Comment (59023g) [inline ui/app/src/components/confirmationDialog/AnimatedConfirmation.vue:52]: necessary?

File: ui/app/src/components/currencyfield/BalanceField.vue [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <script lang="ts">
 import { useCore } from "@/hooks/useCore";
 import { computed } from "@vue/reactivity";
+import { AssetAmount } from "ui-core";
```
Comment (59023g) [inline ui/app/src/components/currencyfield/BalanceField.vue:4]: necc?

File: ui/core/src/actions/clp/index.ts [hunk 3]
```diff
@@ -27,7 +54,10 @@ export default ({
   syncPools();
 
   // Then every transaction
-  api.SifService.onTx(syncPools);
+
+  api.SifService.onNewBlock(async () => {
```
Comment (59023g) [inline ui/core/src/actions/clp/index.ts:58]: What's this for?

File: ui/core/src/actions/peg/index.ts [hunk 4]
```diff
@@ -15,29 +16,33 @@ export default ({
...
-      api.EthbridgeService.lock(cosmosRecipient, assetAmount, 50)
-        .onError((err) => {
-          const payload: any = err.payload;
-          notify({ type: "error", message: payload.message ?? err });
-        })
-        .onComplete(({ txHash }) => {
-          notify({
-            type: "success",
-            message: `Transfer ${txHash} has succeded.`,
+    async lock(assetAmount: AssetAmount) {
```
Comment (59023g) [inline ui/core/src/actions/peg/index.ts:28]: looks good

File: ui/core/src/actions/wallet.ts [hunk 5]
```diff
@@ -32,7 +33,16 @@ export default ({
   };
 
   effect(() => {
-    store.wallet.sif.isConnected = state.connected;
+    if (store.wallet.sif.isConnected !== state.connected) {
```
Comment (59023g) [inline ui/core/src/actions/wallet.ts:36]: I think effects here will be removed for Keplr

File: ui/core/src/api/ClpService/ClpService.test.ts [hunk 6]
```diff
@@ -5,34 +5,17 @@ new WS("ws://localhost:26667/websocket");
...
-  symbol: "cbtk",
-  name: "Banana Token",
-  network: Network.SIFCHAIN,
-});
+const [ROWAN, CATK, CBTK] = getTestingTokens(["ROWAN", "CATK", "CBTK"]);
 
 let service: ReturnType<typeof createClpService>;
 
 beforeEach(() => {
   service = createClpService({
```
Comment (59023g) [inline ui/core/src/api/ClpService/ClpService.test.ts:16]: eventually will want to import config imo

File: ui/core/src/hooks/addLiquidityCalculator.test.ts [hunk 7]
```diff
@@ -1,29 +1,10 @@
...
-  }),
-
-  rowan: Coin({
-    decimals: 18,
-    name: "Rowan",
-    network: Network.SIFCHAIN,
-    symbol: "rowan",
-  }),
-};
+const [ATK, ROWAN] = getTestingTokens(["ATK", "ROWAN"]);
```
Comment (59023g) [inline ui/core/src/hooks/addLiquidityCalculator.test.ts:7]: nice

File: ui/core/src/store/index.ts [hunk 8]
```diff
@@ -1,14 +1,16 @@
...
-import { notifications, NotificationsStore } from "./notifications"
+import { notifications, NotificationsStore } from "./notifications";
+import { LiquidityProvider, Pool } from "../entities";
 export * from "./poolFinder";
 
 export type Store = {
   wallet: WalletStore;
   asset: AssetStore;
   pools: PoolStore;
+  accountpools: { lp: LiquidityProvider; pool: Pool }[];
```
Comment (59023g) [inline ui/core/src/store/index.ts:13]: What's this

File: ui/package.json [hunk 9]
```diff
@@ -34,7 +36,8 @@
     "ganache-cli": "^6.12.0",
     "lerna": "^3.22.1",
     "truffle": "^5.1.48",
-    "vercel": "^21.0.1"
+    "vercel": "^21.0.1",
+    "wait-on": "^5.2.1"
```
Comment (59023g) [inline ui/package.json:40]: what's this

**Review by 59023g — APPROVED** [review]
Couple questions, nits. Otherwise looks good

File: ui/app/src/components/confirmationDialog/AnimatedConfirmation.vue (hunk 1 above)
Comment (ryardley) [reply ui/app/src/components/confirmationDialog/AnimatedConfirmation.vue:52]: Removing. Would be nice to revisit eslint rules and have them fail builds

File: ui/core/src/actions/wallet.ts (hunk 5 above)
Comment (ryardley) [reply ui/core/src/actions/wallet.ts:36]: We have to report connected status somehow. Have you refactored to an event handler?

File: ui/core/src/store/index.ts (hunk 8 above)
Comment (ryardley) [reply ui/core/src/store/index.ts:13]: A bad name for a thing. Basically I need a LiquidityProvider object and a Pool object as they are both related in a 1:1 relationship. I would prefer to refactor core get the names right and do this in a neater way but I have been in a rush.

Comment (ryardley) [reply ui/core/src/store/index.ts:13]: LIquidityProvider has the user's poolUnits on it while the pool has the total units on it

File: ui/package.json (hunk 9 above)
Comment (ryardley) [reply ui/package.json:40]: little cli util that will wait for a url or websocket or file to become available before returning. Used in the `yarn stack` script

File: ui/core/src/api/ClpService/ClpService.test.ts (hunk 6 above)
Comment (ryardley) [reply ui/core/src/api/ClpService/ClpService.test.ts:16]: For tests you mean?

File: ui/core/src/actions/clp/index.ts (hunk 3 above)
Comment (ryardley) [reply ui/core/src/actions/clp/index.ts:58]: tx pool syncing wasn't always occurring so moved to syncing every block for now.

File: ui/core/src/store/index.ts (hunk 8 above)
Comment (ryardley) [reply ui/core/src/store/index.ts:13]: could possibly be `mypools`?

File: ui/core/src/actions/wallet.ts (hunk 5 above)
Comment (59023g) [reply ui/core/src/actions/wallet.ts:36]: No, just async for now

File: ui/core/src/api/ClpService/ClpService.test.ts (hunk 6 above)
Comment (59023g) [reply ui/core/src/api/ClpService/ClpService.test.ts:16]: Yea

**Review by 59023g — APPROVED** [review]

File: ui/core/src/store/index.ts (hunk 8 above)
Comment (ryardley) [reply ui/core/src/store/index.ts:13]: but then they are not "pools"

**Pull request merged**